tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "target": "es2024",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "noEmit": true,
+    "esModuleInterop": true,
+    "module": "esnext",
+    "moduleResolution": "bundler",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "jsx": "preserve",
+    "incremental": true,
+    "plugins": [
+      {
+        "name": "next"
+      }
+    ],
+		"paths": {
```

Comment:
`paths` are not meant to be used like this. https://www.typescriptlang.org/tsconfig/#baseUrl is what this is for :)

Reply:
I just copied what's in the current [jsconfig.json](https://github.com/l2xu/betterbahn/blob/main/jsconfig.json)